train_cora.sh

```diff
@@ -1 +0,0 @@
-python pipeline/main.py --gpu 0 --dataset Cora --lr 1 --input-dropout 0.5 --lamb 5e-3
```

Comment:
The learning rate seems to be too large.

Reply:
See [here](https://github.com/samihaija/mixhop/blob/master/train_cora.sh).